src/Microsoft.PowerShell.ConsoleHost/host/msh/ManagedEntrance.cs

```diff
@@ -109,9 +112,9 @@ public int Start(string consoleFilePath, string[] args)
             try
             {
 #if CORECLR
-                    var banner = ManagedEntranceStrings.ShellBannerNonWindowsPowerShell;
+                var banner = string.Format(ManagedEntranceStrings.ShellBannerNonWindowsPowerShell);
 #else
-                var banner = ManagedEntranceStrings.ShellBanner;
+                var banner = string.Format(ManagedEntranceStrings.ShellBanner);
```

Comment:
string.Format - It seems should be removed.

And why we use Core and non-Core banner?